dynamic_planning_ext/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Dynamic Planning Extension",
+    "author": "Mainframe Monkey BV",
+    "website": "https://www.mainframemonkey.com",
+    "category": "",
+    "version": "14.0.0.0.2",
+    "depends": ["planning", "dynamic_gantt_planning"],
```

Comment:
add a `description` saying "Used for adding the dynamic Gantt view option to the planning it's Gantt view"